packages/common/src/s3.ts

```diff
@@ -1,7 +1,33 @@
-import { S3Client } from '@aws-sdk/client-s3';
+import { GetObjectCommand, S3Client } from '@aws-sdk/client-s3';
+import { createWriteStream } from 'fs';
+import path from 'path';
+import { Readable } from 'stream';
 
-export const getClient = (region: string) => {
+export const getS3Client = (region: string) => {
 	return new S3Client({
 		region,
 	});
 };
+
+export const getFile = async (
+	client: S3Client,
+	bucket: string,
+	key: string,
+	workingDirectory: string,
+) => {
+	try {
+		const destinationPath = `${workingDirectory}/${path.basename(key)}`;
+		const data = await client.send(
+			new GetObjectCommand({
+				Bucket: bucket,
+				Key: key,
+			}),
+		);
+		(data.Body as Readable).pipe(createWriteStream(destinationPath));
```

Comment:
Could we parse with zod to avoid type assertion?

Reply:
Good idea. Made a change for this, does it look OK?

Reply:
thanks! looks great